.changeset/odd-eggs-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
```

Comment:
Probably a patch? 🤔

Reply:
Ah yess, forgot to update that after removing the `floatingOptions`